easybuild/easyblocks/t/trinity.py

```diff
@@ -312,24 +315,53 @@ def sanity_check_step(self):
         """Custom sanity check for Trinity."""
 
         version = LooseVersion(self.version)
-        if version >= LooseVersion('2.0') and version < LooseVersion('2.3'):
-            sep = '-'
-        elif version >= LooseVersion('2.3') and version < LooseVersion('3.0'):
-            sep = "-Trinity-v"
+        if ((version >= LooseVersion('2.0') and version < LooseVersion('2.3')) or
+           (version >= LooseVersion('2.9') and version < LooseVersion('3.0'))):
+            sep = '-v'
```

Comment:
@robqiao For Trinity >= 2.0 < 2.3 it used to be `-`, but now it's `-v`?

Reply:
Thx @boegel for picking this one up, this was a mistake. Fixed now